### DISCUSSION TITLE
Unable to install SO 2.3.10 from ISO

### DISCUSSION THREAD

**Discussion by APT256** [Security-Onion-Solutions/securityonion#2156] [Unsupported Versions] [answered]
Unable to install SO 2.3.10. The iso image matches the posted hash, so I know that the download is not corrupted. I am using balenaEtcher to write it to a 64GB flash drive and I have it set to do a verification after write. This comes back w/o any issues. I am attempting to install i t on a Dell OptiPlex 5050 micro with a quad-core i7 and 32Gb of RAM. I have used this system with previous version of SO and did not have any issues with those. I have tried all three installation options and they all fail the same way. At the beginning, I seethe following errors:
dracut-pre-udev[353]: modprobe: ERROR: could not insert 'floppy': No such device
usb usb2-port4: Cannot enable. Maybe the USB cable is bad?
usb usb2-port4: cannot disable (err = -32)
...
dracut-initqueue[#####]: Warning: dracut-initqueue timeout - starting timeout scripts
... a bunch more of these same errors with a different number ...
dracut-initqueue[#####]: Warning: Could not boot.
dracut-initqueue[#####]: Warning:  /dev/root does not exist

The installer then boots into the dracut emergency shell
[rdsosreport.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5623400/rdsosreport.txt)

Any help resolving this will be greatly appreciated!

v/r
Jon

**Comment by dougburks** [comment, marked as answer]
Have you tried changing the machine from BIOS to UEFI (or vice-versa)?

If all else fails, you should be able to install standard CentOS 7 or Ubuntu 18.04 and then install on top of that:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

**Reply by APT256** [reply]
Thanks for the suggestion Doug, I had not thought of that. I will try changing to UEFI first, and if that doesn’t work, I will do a base Ubuntu install followed by installing SO  on top of it. I will post back here with my UEFI results in case it might help someone else.

Jon

**Reply by APT256** [reply]
I tried all 3 boot options: 1) Legacy Boot Mode 2) UEFI - Secure Boot Off 3) UEFI - Secure Boot On. All three boot option ended in the same error. I am going to try and install CENTOS and install over the top next.

**Reply by APT256** [reply]
This appears to be a CENTOS issue. I have the exact same errors when trying to install CENTOS 7. However, I have no issues at all installing Ubuntu, so I will attempt to setup SO on Ubuntu 18.04.
